src/test/kotlin/net/atos/zac/productaanvraag/ProductaanvraagServiceTest.kt

```diff
@@ -613,6 +613,10 @@ class ProductaanvraagServiceTest : BehaviorSpec({
                             "inpBsn" to behandelaarBsn,
                             "rolOmschrijvingGeneriek" to "behandelaar"
                         ),
+                        mapOf(
+                            "inpBsn" to behandelaarBsn,
+                            "roltypeOmschrijving" to "Behandelaar"
```

Comment:
Is this not the same as the one above? Or should it be unique items, just with other capitalization? If so, an explanation will help clear that up here.

Reply:
We test `rolOmschrijvingGeneriek` and `roltypeOmschrijving` here. They both can be specified and contain Behandelaar